docs/sphinx/user_guide/feature/view.rst

```diff
@@ -92,21 +92,57 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+Reshape
+^^^^^^^
+The ``RAJA::Reshape`` method is designed as a simplified multi-dimensional pointer abstraction
+only requiring the pointer, dimensions, and prefered layout type based on a template specilization.

```

Comment:
This seems too focused on how its implemented via specialization than how its used. Could this just say that you need to provide a pointer, dimensions, and a layout type?

Reply:
Sure, I think we can make that works. I coined the term meta-layout as predefined layout types as suggested.